.buildkite/pipeline.yml

```diff
@@ -54,7 +54,7 @@ steps:
       docker-compose#v2.5.1:
         run: app
     <<: *common
-  - label: "Integration (collector, m3em, agg)"
+  - label: "Integration (collector, m3em, agg, msg)"
```

Comment:
do you want to add `test-ci-integration-msg` too

Reply:
sure